models/bonkbot/solana/bonkbot_solana_trades.sql

```diff
@@ -0,0 +1,152 @@
+{{ config(
+    alias = 'trades',
+    schema = 'bonkbot_solana',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['blockchain', 'tx_id', 'tx_index', 'outer_instruction_index', 'inner_instruction_index', 'token_bought_address', 'token_sold_address'],
+    post_hook='{{ expose_spells(\'["solana"]\',
+                                "sector",
+                                "dex",
+                                \'["whale_hunter"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2023-08-17' %}
+{% set fee_receiver = 'ZG98FUCjb8mJ824Gbs6RsgVmr1FhXb2oNiJHa2dwmPd' %}
+{% set wsol_token = 'So11111111111111111111111111111111111111112' %}
+
+WITH
+  allFeePayments AS (
+    SELECT
+      tx_id,
+      IF(balance_change > 0, 'SOL', 'SPL') AS feeTokenType,
+      IF(
+        balance_change > 0,
+        balance_change / 1e9,
+        token_balance_change
+      ) AS fee_token_amount,
+      IF(
+        balance_change > 0,
+        '{{wsol_token}}',
+        token_mint_address
+      ) AS fee_token_mint_address
+    FROM
+      {{ source('solana','account_activity') }}
+    WHERE
```

Comment:
as mentioned in discord, please add incremental filters
examples in the `docs/`